text/0092-logs-vision.md

```diff
@@ -0,0 +1,128 @@
+# OpenTelemetry Logs Vision
+
+The following are high-level items that define our long-term vision for 
+Logs support in OpenTelemetry project, what we aspire to achieve.
+
+This a vision document that reflects our current desires. It is not a commitment
+to implement everything precisely as listed. The primary purpose of this
+document is to ensure all contributors work in alignment. As our vision changes
+over time maintainers reserve the right to add, modify, and remove items from
+this document.
+
+(The content in this proposal is a starting point for discussion. I believe it
+is important for us to collectively come up with such vision. I am posting
+this PR largely to open the discussion. Comments, thoughts and opinions are
+highly welcome!)
+
+This document uses vocabulary introduced in https://github.com/open-telemetry/oteps/pull/91
+
+### First-class Citizen
+
+Logs are first-class citizen in observability, along with traces and metrics.
+We will aim to have best-in-class support for logs at OpenTelemetry.
+
+### Correlation
+
+OpenTelemetry will define how logs will be correlated with traces and metrics
+and how this correlation information will be stored.
+
+Correlation will work across 2 major dimensions:
+- To correlate telemetry emitted for the same request (also known as Request
+  or Trace Context Correlation),
+- To correlate telemetry emitted from the same source (also known as Resource
+  Context Correlation).
+
+### Logs Data Model
+
+We will design a Log Data model that will aim to correctly represent all types
+of logs. The purpose of the data model is to have a common understanding of what
+a log record is, what data needs to be recorded, transferred, stored and
+interpreted by a logging system.
+
+Existing log formats can be unambiguously mapped to this data model. Reverse 
+mapping from this data model is also possible to the extent that the target log 
+format has equivalent capabilities.
+
+We will produce mapping recommendations for commonly used log formats.
+
+### Log Protocol
+
+Armed with the Log Data model we will aim to design a high performance protocol
+for logs, which will pursue the same [design goals](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/proto-design-goals.md)
+as we had for traces and metrics protocol design.
+
+Most notably the protocol will aim to be highly reliable, have low resource
+consumption, be suitable for all participant nodes, ensure high throughput,
+allow backpressure signalling and be load-balancer friendly.
+
+The reason for this design is to have a single protocol that can deliver
+logs, traces and metrics via one connection and satisfy all design goals
+(as far as we know this does not exist today).
+
+### Unified Collection
+
+We aim to have high-performance, unified Agent and Collector that support
```

Comment:
```suggestion
We aim to have a high-performance, unified Agent and Collector that supports
```

It would also make sense to link to the glossary where the terms "agent" and "collector" are explained.

Reply:
Removed "Agent" (not needed in this context) and added a link to "Collector".